src/components/AccountSelector.test.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import selectEvent from 'react-select-event';
+
+import { simpleRender, screen, fireEvent } from 'test-utils';
+import { fAccounts } from '@fixtures';
+import { sortByLabel } from '@utils';
+import AccountSelector from './AccountSelector';
+
+type Props = React.ComponentProps<typeof AccountSelector>;
+
+function getComponent(props: Props) {
+  return simpleRender(
+    <form role="form">
+      <AccountSelector {...props} />
+    </form>
+  );
+}
+
+const defaultProps = {
+  accounts: fAccounts,
+  name: '',
+  value: sortByLabel(fAccounts)[0],
+  onSelect: jest.fn()
+};
+
+describe('AccountSelector', () => {
+  test('it displays account info by default', async () => {
+    getComponent(defaultProps);
+    const accounts = sortByLabel(defaultProps.accounts);
+    // Displays the account label
+    expect(screen.getByText(new RegExp(accounts[0].label, 'i'))).toBeDefined();
```

Comment:
Is there a reason you don't use the `getByText` returned by `getComponent`?

(Goes for everywhere in the PR)

Reply:
It's from https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen
> The benefit of using screen is you no longer need to keep the render call destructure up-to-date as you add/remove the queries you need. You only need to type screen. and let your editor's magic autocomplete take care of the rest.